### ISSUE TITLE
[0.5.4] Negative RF flowing out of too long transmission line due to transmission losses

### ISSUE THREAD

**Issue by dantedarkstar** [BluSunrize/ImmersiveEngineering#374]
I was testing whether power lines can go across unloaded chunks and built a quite long line (~400 blocks). On the end I was getting negative RF accumulating in the capacitor I attached (which I assume should not happen). I did not investigate further what happens to negatively-charged capacitor (I was planning to set loss to 0 in config anyway, which of course removed the issue).

IE 0.5.4, inside Horizons 2 Daybreaker FTB pack 1.0.2 (unmodified, except for one mod - Computronics, removed)

**Comment by theunkn0wn1** [comment]
This is due to the power loss calculations currently being... less than desireable. - even @BluSunrize is not happy with it atm - HV simply cannot go that far. You can go exactly  320 meters (20 chunks) at 100% power loss, leaving the affective range 130 meters, or 10 chunks.

In case you are wondering how i came to this number? We are currently discussing this topic in #362

**Comment by cobra** [comment]
It's still happening in e6bf7a3, but any change to the affected capacitor or a connector attached to an output reset the charge level to 0, so this shouldn't do any harm or enable exploits (derived from playing around, not from examining code).

The problem is that the loss rate can get bigger than 1.0 which breaks any subsequent calculations, ultimately losing more energy than is being sent, and nothing ever stops adding a negative amount of energy (which is quite interesting on its own). I've fixed that but I'm still investigating some other weird behaviour I found during testing to avoid breaking stuff or missing even more brokenness. There are some places where wrong values could get passed along unchecked.

I'm also waiting for @malte0811's energy net fixes for further testing and integration. Additionally, having to check and rewire a couple of hundred blocks due to lost connections after a restart gets annoying really fast.
